makefile

```diff
@@ -15,3 +15,14 @@ itest:
 
 build:
 	pyb
+
+run:
+	python src/main/scripts/run.py install
```

Comment:
Why the `install` argument? It's not used in the `run.py` script anywhere at all.